Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/model/orm/Loadable.java

```diff
@@ -234,9 +236,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "Loadable{id=" + id + ", mode=" + mode + ", board='" + boardCode + '\'' + ", no=" + no + ", title='"
-                + title + '\'' + ", listViewIndex=" + listViewIndex + ", listViewTop=" + listViewTop + ", lastViewed="
-                + lastViewed + ", lastLoaded=" + lastLoaded + ", markedNo=" + markedNo + ", dirty=" + dirty + '}';
+        return "Loadable{id=" + id + ", mode=" + mode + ", board='" + boardCode + '\'' +
+                ", no=" + maskPostNo(no) + '\'' + ", listViewIndex=" + listViewIndex +
+                ", listViewTop=" + listViewTop + ", lastViewed=" + lastViewed + ", lastLoaded=" +
+                maskPostNo(lastLoaded) + ", markedNo=" + maskPostNo(markedNo) + ", dirty=" + dirty + '}';
```

Comment:
If you wouldn't mind, add in the loadable downloading state to this String. Seems it was missed out when thread downloading was added.

Reply:
Done.